### ISSUE TITLE
Requesting key... never completes for CD key

### ISSUE THREAD

**Issue by jvarho** [ValveSoftware/steam-for-linux#212]
I'm trying to get the CD key link for a non-Linux game I own (Crusader Kings II), so I can use it to sign up on their forums. The request never completes, it just says "Requesting key..." and copying to clipboard does nothing.

1) Open the game in library.
2) Click the CD key link on the right hand side.
3) A new window pops up but the request never completes.

System info:

Processor Information:
    Vendor:  AuthenticAMD
    Speed: 2800 Mhz
    4 logical processors
    4 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Supported
    SSE4a:  Supported
    SSE41:  Unsupported
    SSE42:  Unsupported

Network Information:
    Network Speed:  

Operating System Version:
    Ubuntu 12.10 (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.5.0-21-generic
    X Server vendor:  The X.Org Foundation
    X Server release:  11300000

Video Card:
    Driver:  X.Org Gallium 0.4 on AMD RV770

```
Driver Version:  2.1 Mesa 9.0
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 60 Hz
VendorID:  0x1002
DeviceID:  0x9442
Number of Monitors:  1
Number of Logical Video Cards:  1
Primary Display Resolution:  1920 x 1080
Desktop Resolution: 1920 x 1080
Primary Display Size: 20.08" x 11.30"  (23.03" diag)
                                        51.0cm x 28.7cm  (58.5cm diag)
Primary VRAM Not Detected
```

Sound card:
    Audio device: Nvidia MCP77/78 HDMI

Memory:
    RAM:  3954 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.UTF-8
    Microphone:  Not set
    Total Hard Disk Space Available:  24605 Mb
    Largest Free Hard Disk Block:  16279 Mb

Installed software:

Recent Failure Reports:
    Fri Dec 21 11:53:12 2012 GMT: file ''/tmp/dumps/assert_20121221135304_1.dmp'', upload yes: ''CrashID=bp-20540da8-691f-4d9d-be7c-a727e2121221''

**Comment by jared-wallace** [comment]
Did this ever get fixed? I'm seeing this with the latest build. Trying to get my Doom 3 key.

**Comment by Strit** [comment]
@jared-wallace I have the exact same issue. Purchased Doom 3 pack, got the doom3-data-steam and doom3 packages from the AUR installed. But I can't get the CD key from steam, which I need to activate the game. The game launches fine, but I get "stuck" on the activation step.

Any idea what can be done here?

**Comment by jared-wallace** [comment]
I just started downloading the game on a Windows box. VM would probably work too. As soon as it started, I could grab the key. 

On December 11, 2015 9:04:40 AM CST, Dan Johansen <notifications@github.com> wrote:
>@jared-wallace I have the exact same issue. Purchased Doom 3 pack, got
>the doom3-data-steam and doom3 packages from the AUR installed. But I
>can't get the CD key from steam, which I need to activate the game. The
>game launches fine, but I get "stuck" on the activation step.
>
>Any idea what can be done here?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ValveSoftware/steam-for-linux/issues/212#issuecomment-163958262

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Strit** [comment]
I really hope that "get a windows install" is not the answer from Valve to this problem.

**Comment by simfoo** [comment]
I'm having the same problem. Bought Trackmania 2 Stadium from Steam on Linux but cannot get the CD key to use it to play the game using Wine. The dialog opens but is stuck at "Requesting key..."

**Comment by rea987** [comment]
Hi, I am copying my findings from #5024.

#### System Information

* Steam client version: Jun 8 2017, at 04:34:18; Steam API: v017; Steam package versions: local
* Distribution: Ubuntu Mate 17.04 x64
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### The issue in as much detail as possible:

Steam client has an option to show **CD Key** of certain games. Here are those games:

ShowCDKeyOnLaunch: https://steamdb.info/search/?a=app_keynames&keyname=179
ShowCDKeyInMenu: https://steamdb.info/search/?a=app_keynames&keyname=181

However, as I found out, **Steam for Linux** is unable to show cd keys of non-Linux titles:

Here is the CD Key output of [Warhammer 40,000: Dawn of War - Game of the Year Edition](http://store.steampowered.com/app/4570/Warhammer_40000_Dawn_of_War__Game_of_the_Year_Edition/):

- Steam for **Linux**: https://yadi.sk/i/XgOE57633K3cgM
- Steam for **Windows** via PlayOnLinux: https://yadi.sk/i/1XjAUZ4l3K3cgY

When I try to see the cd key of a Linux game, Steam for Linux instantly shows the key:

[Cities in Motion 2](http://store.steampowered.com/app/225420/Cities_in_Motion_2/) via Steam for Linux:

https://yadi.sk/i/MZ9wymlo3K3d8T

As it turned out Steam client two different method to show CD keys; **ShowCDKeyOnLaunch** and **ShowCDKeyInMenu**. Apparently, CD key "**option**" of the games that use ShowCDKeyOnLaunch method does not show up on Steam client if the games don't support that OS (eg Linux). [FlatOut: Ultimate Carnage](store.steampowered.com/app/12360/FlatOut_Ultimate_Carnage/) which uses ShowCDKeyOnLaunch method is an example of that:

Steam for Linux: https://yadi.sk/i/6lVS9Hjx3K5AiM
Steam for Windows via PlayOnLinux: https://yadi.sk/i/2NZHmMaP3K5Ai5

On the other hand, CD key option of the games that use ShowCDKeyInMenu method does appear on Steam client of all 3 operating systems, but CD key cannot be retrieved if the game does not support the operating system in question (eg Linux). Previously showed [Warhammer 40,000: Dawn of War - Game of the Year Edition](http://store.steampowered.com/app/4570/Warhammer_40000_Dawn_of_War__Game_of_the_Year_Edition/) is an example of that.

A [Feral Interactive](https://www.feralinteractive.com) employee, [edddeduck_feral](https://www.reddit.com/user/edddeduck_feral) explained the cause of the problem on Reddit:

https://www.reddit.com/r/linux_gaming/comments/6grmop/steam_for_linux_doesnt_show_cd_key_for_windows/distlgd/

#### Steps for reproducing this issue:

1. Launch Steam for Linux
2. Find a game which has [CD key](https://steamdb.info/search/?a=app_keynames&keyname=179) [option](https://steamdb.info/search/?a=app_keynames&keyname=181).
3. See if CD key option appears.
4. If CD key option is present, click it to get license code of non-Linux games.

. My system information via Steam client: [Github Gist](https://gist.github.com/rea987/fe150c69a717a8296e5faab25a3f5522)

**Comment by jomarocas** [comment]
The solution on valve is, games execute proton games only for windows or mac, put the same key that have in windows

**Comment by Rich43** [comment]
I have having this issue with Trainz Simulator 12, checking for the key is mandatory on startup, so the game will not start.

**Comment by Bednar87** [comment]
I have the same issue when installing assassin's creed unity.

**Comment by egon2003** [comment]
I also have this issue with Assassins Creed Unity

**Comment by abiriukov** [comment]
I have the same problem with AC: Unity. I've tried both my copy of the game and my wife's copy, but neither works...

**Comment by EyeOfMidas** [comment]
I am unable to play Assassin's Creed II for the same reason. I have a CD key I got from Steam Support, but I can't get past this step to actually use it.

**Comment by WoutLin** [comment]
I have this issue with Driver San Francisco appid 33440.  I had Steam on PlayOnLinux and actually got a cd key from Steam.  However my PlayOnLinux got deleted and I wish to use Steam Proton but keep getting this error.  Even though I have a cd key (I copied it) there is no way (that I have found) to install the key manually.  I contacted Steam Support and the response was to report it here.

**Comment by jomarocas** [comment]
im waiting for this support for proton in linux @kisak-valve but nothing for update, the game that i test is assassin creed 2 only

**Comment by GabrielGMartinsBr** [comment]
I have the same problem. When I try to run the game and the CD Key error message appears, these two lines are inserted into the terminal:

_GameAction [AppID 33230, ActionID 1] : LaunchApp changed task to ShowCDKey with ""
GameAction [AppID 33230, ActionID 1] : LaunchApp waiting for user response to ShowCDKey ""_

**Comment by WoutLin** [comment]
I installed Steam on PlayOnLinux and now Driver San Francisco gets the cd key but through Steam Proton it still will not.  The only problem now is when the game is launched it says "running" but after about a minute with nothing happening "running" stops and nothing.

**Comment by tijder** [comment]
I got a way to bypass the steam key not loading. For Assassin's Creed Brotherhood I got it fixed with this steps:
You need to edit ~/.steam/steam/appcache/appinfo.vdf . Edit the line with the text "legacykeyregistrylocation^@HKEY_CURRENT_USER\Software\Valve\TestApp48195\SteamKey" edit it to "legacykeyregistrylocation2^@HKEY_CURRENT_USER\Software\Valve\TestApp48195\SteamKey". With this typo steam will start the game without loading the game key.

**Comment by EyeOfMidas** [comment]
Using @tijder 's tip, I was able to get Assassin's Creed II to start.

I used `nano` to edit the key
`legacykeyregistrylocation^@HKEY_CURRENT_USER\Software\Valve\TestApp33361\SteamKey`
and renamed it to 
`legacykeyregistrylocation2^@HKEY_CURRENT_USER\Software\Valve\TestApp33361\SteamKey`

I was prompted to sign in to UPlay and had to link my steam account, but once I did I was able to start Assassin's Creed II without any trouble.

**Comment by tijder** [comment]
The problem is Steam will repair the file. So I think a nice generic command to run before starting Steam is something like this:
`sed -i 's/legacykeyregistrylocation/legacykeyregistrylocation2/g' ~/.steam/steam/appcache/appinfo.vdf`

**Comment by abiriukov** [comment]
I've tried the solution @tijder suggested, but it doesn't work in my case.
I've noticed that the Steam client recreates the file every time it starts. Then, if I edit the file with the client open it doesn't produce any effect. Maybe I'm doing something wrong...
Besides this solution wouldn't work for AC: Unity, because it doesn't seem to store the key in the registry so there is no "legacykeyregistrylocation" record.

**Comment by jomarocas** [comment]
> The problem is Steam will repair the file. So I think a nice generic command to run before starting Steam is something like this:
> `sed -i 's/legacykeyregistrylocation/legacykeyregistrylocation2/g' ~/.steam/steam/appcache/appinfo.vdf`

for me working goog but, i waiting for this solution in steam

**Comment by Arucard1983** [comment]
Some games from Assassin's Creed saga are not playable on Proton due to CDKey failure. Using Lutris and the Windows version of Steam, all games that requires CDKey runs fine.

Valve should fix this annoyance quickly.

My sugestion are adding a fake Windows setting, such as PROTON_FORCE_WINDOWS=1 on relevant games, in order to get the Windows version keys, and resume the Steam DRM checks.

**Comment by PedroHLC** [comment]
[Came here from #68, took a look at #190. Someone should close them]
Here @tijder solution isn't working, I have 48196 (ACIIB Deluxe CD Key) and 48198 (ACIIB Deluxe Mac) and changing them both doesn't seem to solve anything, using the generic sed doesn't either.

**Comment by antimech** [comment]
@PedroHLC I confirm. @tijder it doesn't work now.

**Comment by tijder** [comment]
@antimech for me it's still working. Maybe something else needs to be changed in this file ~/.steam/steam/appcache/appinfo.vdf. I'm running the normal version of ac:brotherhood.

**Comment by DominykasPetke** [comment]
The editing appinfo.vdf file trick doesn't seem to be working anymore in the Beta (2020-03-03), stable is still fine. (AC:Brotherhood)

**Comment by Alstruit** [comment]
> The problem is Steam will repair the file. So I think a nice generic command to run before starting Steam is something like this:
> `sed -i 's/legacykeyregistrylocation/legacykeyregistrylocation2/g' ~/.steam/steam/appcache/appinfo.vdf`

Make sure to set the immutable bit to prevent steam from reverting it back.

`# chattr +i ~/.steam/steam/appcache/appinfo.vdf`

**Comment by NexonSU** [comment]
> > The problem is Steam will repair the file. So I think a nice generic command to run before starting Steam is something like this:
> > `sed -i 's/legacykeyregistrylocation/legacykeyregistrylocation2/g' ~/.steam/steam/appcache/appinfo.vdf`
> 
> Make sure to set the immutable bit to prevent steam from reverting it back.
> 
> `# chattr +i ~/.steam/steam/appcache/appinfo.vdf`

Stable 27.04.2020 not working =(

**Comment by mohkale** [comment]
still not working with AC2 for me :cry:.

**Comment by Aquassaut** [comment]
Same with AC2 and AC Brotherhood

**Comment by jayrlsw** [comment]
I am also having this issue with AC Brotherhood

**Comment by Cheaterman** [comment]
Bumping this for AC Brotherhood - seems like ValveSoftware/Proton#190 is also a duplicate of this.

**Comment by jomarocas** [comment]
a update for this?

**Comment by vaslabs** [comment]
having the same issue, only managed to play by installing through lutris installing steam dxvk

**Comment by cverstege** [comment]
This bug is now known for nearly 8 years. And it most probably is quite an easy fix inside the Steam client. This issue is probably making dozens of games unplayable through proton which otherwise would just work fine. The devs even went the extra mile to prevent the old workaround (this might have been a side effect of something else, but still...)
Valve time is crazy.
This issue will probably be fixed once Portal 3 or L4D3 is released.

Please Valve devs, I know you hate it too, when a bug this old is still around :) And I'd love to play through the whole Assassin's Creed Saga on my Linux machine!

**Comment by cverstege** [comment]
To provide some more Information:
Assassin's Creed 2 and Assassin's Creed Brotherhood won't launch because the key "can't be retrieved from steam". Assassin's Creed Revelations works just fine through Proton.

I think this might be, because of some misconfiguration with the steam depots. AC2 and AC Brotherhood are both available on Mac, but via a strange Mac DLC? Normally this would be done with via a Mac depot on the steam servers, but here it seems to be done with this DLC.
So the DLCs for me AC2 are:
* Assassin's Creed 2 - Mac
* Assassin's Creed II - Deluce CD Key

and for AC Brotherhood:
* Assassin's Creed Brotherhood - Standard CD Key
* Assassin's Creed Brotherhood - Standard CD Key Mac

inside `~/.steam/steam/appcache/appinfo.vdf` the games and DLCs have a `validoslist` which might be the cause for this? I'm just guessing here, as other games that are Windows only and have keys and are launching just fine. Their keys are retrievable as well. Like Assassin's Creed Revelations or the Trackmania² games and Trackmania Turbo.

Maybe @kisak-valve @Plagman or @frankc-valve can have a look at this again, as this might be a different cause than the original bug report had.

FYI:
`sed -i 's/validoslist/oslist/g' ~/.steam/steam/appcache/appinfo.vdf` did not help when trying to launch AC2 or trying to retrieve the key, But I think that the file will be overwritten when launching steam. I did not investigate it further.
Maybe someone else can try an pick up from here.

**Comment by vaslabs** [comment]
@cverstege I think you are right about steam client getting confused with the keys.
If you run it on command line steam does
[33230]Non-Steam Controller Configs Enabled: 1
ExecuteSteamURL: "steam://cdkeys/33230"

but if you run `steam steam://cdkeys/33361 it works`

33361 is the windows dlc

**Comment by cverstege** [comment]
Thanks @vaslabs for the hints.
I think the problem is the `ShowCDKeyOnLaunch` key for the Mac DLCs, as the Windows keys can be requested just fine. I tested it as well.
This flag (see https://steamdb.info/app/33362/ for AC2 and https://steamdb.info/app/48197/ for ACB) forces Steam to fetch the CD key when launching the game, even though were not on macos. This flag is also set on the Windows Key, but this seems to be fine.
I need to test if the Mac keys are fetchable from a Windows install or what exactly happens there
We could remove this from `~/.steam/steam/appcache/appinfo.vdf` but Steam will overwrite this which each launch. Maybe starting steam in offline mode will prevent an overwrite? I can try and test this tomorrow.

Trying to add a fake key for the mac dlc in `~/.steam/registry.vdf` does not seem to work.

**Comment by cverstege** [comment]
In Windows the Mac key is not retrievable as well. Steam is tuck requesting the key. The key window opens, but requesting the key won't finish. On linux the key windows won't even open and the error message to try again in a few minutes pops up immediatly.

On Windows the Mac DLCs won't even install/are not visible at all. So on Windows the Steam client only installs the Windows Key DLC (33361) and doesn't even list the Mac Key DLC (33362) as available.

So this is acutally not a bug when requesting the CD key, but rahter a SteamPlay bug, where Steam on Linux makes the Mac Only DLC available as well, even though we do only want the Windows version to be installed and to be used with Proton!

**Comment by vaslabs** [comment]
It looks like a steam play issue, even if I do
```
WINEPREFIX=~/.steam/steam/steamapps/compatdata/33230/ ~/.steam/steam/steamapps/common/Proton\ 5.13/dist/bin/wine AssassinsCreedIIGame.exe
```
it still goes for the key

**Comment by Cheaterman** [comment]
Valve, please fix.

For what it's worth, I tried all the workarounds and the binary hacking I could on the appinfo.vdf, also using SteamEdit (which does _not_ list the "Brotherhood Mac Key" DLC, or other DLCs in general?) to fix the checksum after making manual changes. I tried to set showcdkeyonlaunch to 0 (first only on the Mac DLC, then on both), no success. I tried the legacykeyregistrylocation thing, no effect. I tried adding is_hidden to steam_edit dict, no success - file becomes unable to load in SteamEdit so I'm sure enough it wouldn't load in proper Steam - I suspect the size of the dict is stored somewhere in the header given the diffs I observed. I tried manually requesting the key - and I've got my key, but I tried adding it to registry.vdf, no success.

I want to play this game please.

Valve, please fix.

**Comment by cverstege** [comment]
@Cheaterman check out https://github.com/ValveSoftware/Proton/issues/190#issuecomment-734984986
SteamEdit does list DLCs, you need to toggle them in the bar at the top right. There is a filter where you can turn on/off games, dlcs, movies, software etc.
![SteamEdit DLCs](https://user-images.githubusercontent.com/6241018/101242712-730f3600-36fb-11eb-9d82-c959b29d6e28.png)
Do you see the green switch at the list on the top right. That's for DLCs. I did turn everything else off, that's why the buttons are grey. Then find the Brotherhood Mac DLC and set it to hidden for the game to launch.
You also need to disable the steam overlay. Otherwise steam will crash when launching the game. The game will be running fine, but steam won't track your playtime. See #7494.

**Comment by Cheaterman** [comment]
Hahaha, thanks a lot @cverstege !!! I'm sorry to be that dumb, I didn't realize these icons were filters, I thought they were attibutes of whatever we had selected! I assure you that I tried hard, hahaha.

I confirm the key shows successfully when the file is edited through SteamEdit, stuff is installing right now, I'll confirm if I need to disable Steam overlay in an edit.

Thanks a lot again, sorry it wasn't clear for me :-)

**Comment by Cheaterman** [comment]
With the brand new update to Proton, the game crashes to desktop again. Probably not the right place for it, I'll look for the appropriate issue.

**Comment by vaslabs** [comment]
I tried the steam edit suggestion and it worked !!! Thanks @cverstege .
To run steam edit easily on Linux:
Create a new wine prefix e.g ~/.steamedit with wine tricks
Create a symbolic link in drive_c/Program Files (x86)/Steam that points to ~/.steam/steam , install dotnet461 with winetricks ( I think dotnet 5 also works) . Then run WINEPREFIX=~/.steamedit wine SteamEdit.exe

**Comment by mendiromania** [comment]
please @kisak-valve fix this I do not have windows to check the cd keys and running a 3rd party app is just meh, I'll do it this time but it's meh.

**Comment by Cheaterman** [comment]
I don't have Windows either but TBF the SteamEdit thing is good enough.
Besides it's probably out of Valve's control so please be nice to Kisak 😅

>

**Comment by mendiromania** [comment]
kisak is cool but showing cd keys for a game i paid for is totally under valve's control

**Comment by Cheaterman** [comment]
Perhaps you are right, yes. Re-reading the context, it does seem to be the
consensus, but with a functioning workaround I can see how this is low
priority on their TODO list 🙂

**Comment by mendiromania** [comment]
It doesn't sound that hard to implement (the feature works for linux games, I guess there's a check wether it's a platform game or not).

**Comment by mendiromania** [comment]
the option now appers under manage > cd keys, but it fails to retrieve the cd keys with message:
Failed to request product key from Steam, please try again in a few minutes

**Comment by rileyrg** [comment]
AC Odyssey there is no cdkeys undermanage.Where is it? Im tired f fighting with uplay.

**Comment by smintman** [comment]
Please please Valve can you resolve this we shouldn't need to hacking the files to get this to work, just correct the client incorrectly trying to request the Mac DLC!?!?

**Comment by ell1e** [comment]
This still seems to be broken, around 9 years later. Slightly baffling. What's the hold-up?

**Comment by jeifour** [comment]
Please Valve I want to get my CD key for the game I bought. Doesn't matter if it's a Windows game and I'm on Mac.

**Comment by PolarBass0511** [comment]
Hello, I was told by "Charles" to post in this specific thread.  This is in regards to my current ticket in with Steam Support (Ticket: HT-5XTF-6F3Y-DWVW).  I run PoP!_OS on an AMD/ATI machine, and reference ProtonDB with every game I play.  Although I have not had a problem with Assassin's Creed I, II, Odyssey, or Origins, AC Brotherhood and Revelations refuse to let me play.  I will click play, and then receive an error that Steam failed to retrieve or request an activation key.  Using the Manage > CD Keys option pulls the same error.  Either way, I can go no further in attempting to play the game(s) until this is resolved.  I even downloaded the Ubisoft Connect client, and can successfully run it through Steam client via Proton 6.3-8, but this game does not show, and the key I received from "Charles" today was read as not valid.  Many thanks for any help you can provide, as I would love to continue the AC story, support Steam, and not feel like I was cheated out of money that could have been better used as a student loan payment.

**Comment by smintman** [comment]
@PolarBass0511 I wrote a script to get AC 2 and Brotherhood working. I've not had an issue with getting CD key for revelations so not added it but maybe could help you with brotherhood

https://github.com/smintman/assfix

**Comment by begin-theadventure** [comment]
https://github.com/begin-theadventure/acfix
I created another way to apply the fix for Assassin's Creed, which I think is better because you can add it to multiple ACs at once - practically unlike with .vdf - theoretically well unless you wanna share your entire library...
+I added fixes for Unity and Brotherhood Standard Edition.

**Comment by tijder** [comment]
It looks like this bug is finally solved! Today I could start AC:II and AC:Brotherhood without any problems. Steam even showed the cd key! :tada:

**Comment by smintman** [comment]
I managed to load Assassins creed 2 deluxe and Brotherhood without issue too! (well ubisoft connect crashes after update, but second try works fine) 
Hopefully this is resolved now.... only took 10 years!!!

Can others give it a go and check?

**Comment by cverstege** [comment]
Seems to be fixed in the current Steam Beta (2nd April 2022). The normal Client still does not work.

**Comment by begin-theadventure** [comment]
> It looks like this bug is finally solved! Today I could start AC:II and AC:Brotherhood without any problems. Steam even showed the cd key! tada

I forgot to reply after this comment but it does indeed work now for AC: II, Brotherhood & Unity.

**Comment by peldax** [comment]
AC2 is still not working for me. I am getting following prompt:
![Screenshot_2022-04-26_11-59-12](https://user-images.githubusercontent.com/10790033/165551594-26045d56-78cb-4868-8be8-24e41399f0cf.png)
I already played the game on windows few years ago, so its kinda weird that my key is not attached to my account already.

**Comment by smintman** [comment]
> AC2 is still not working for me. I am getting following prompt:
> ![Screenshot_2022-04-26_11-59-12](https://user-images.githubusercontent.com/10790033/165551594-26045d56-78cb-4868-8be8-24e41399f0cf.png)
> I already played the game on windows few years ago, so its kinda weird that my key is not attached to my account already.

Are you using the beta steam client?

**Comment by EyeOfMidas** [comment]
I just enabled the Steam Beta Client, restarted Steam and played Assassin's Creed II for 10 minutes. It works perfectly. Can't wait for this to be in the main release and we can finally close this 10-year-old issue. 👍

**Comment by peldax** [comment]
> > Replying to [#212 (comment)](https://github.com/ValveSoftware/steam-for-linux/issues/212#issuecomment-1111127247)
> 
> Are you using the beta steam client?

I just updated to Steam Beta and its working. Thanks!

**Comment by kisak-valve** [comment]
Closing as fixed in the [2022-05-12](https://store.steampowered.com/oldnews/138198) Steam client update (uncredited).
